amdgpu-clocks

```diff
@@ -167,21 +182,6 @@ function set_custom_states() {
     echo "${VDDGFX_OFFSET}" > "${SYS_PP_OD_CLK}"
   fi
   echo 'c' > "${SYS_PP_OD_CLK}"
-  if [ "${FORCE_LEVEL}" ]; then
-    echo ${FORCE_LEVEL} > ${PWR_LEVEL}
-  fi
-  if [ "${FORCE_PROFILE}" ]; then
-    echo ${FORCE_PROFILE} > ${PWR_PROFILE}
-  fi
-  if [ "${FORCE_SCLK}" ]; then
-    echo "${FORCE_SCLK}" > "${SYS_DPM_SCLK}"
-  fi
-  if [ "${FORCE_MCLK}" ]; then
-    echo "${FORCE_MCLK}" > "${SYS_DPM_MCLK}"
-  fi
```

Comment:
The `FORCE_SCLK` and `FORCE_MCLK` must not be moved here, they gotta stay before commit (`echo 'c' > "${SYS_PP_OD_CLK}"`), otherwise the change would break older cards.